bundles/org.openhab.core.voice/src/main/java/org/openhab/core/voice/internal/DialogProcessor.java

```diff
@@ -160,8 +163,11 @@ private void abortKS() {
     private void closeStreamKS() {
         AudioStream stream = streamKS;
         if (stream != null) {
-            // Due to an issue in JavaSoundAudioSource ( https://github.com/openhab/openhab-core/issues/2702 )
-            // we do not try closing the stream
+            try {
+                stream.close();
+            } catch (IOException e) {
+                logger.warn("IOException closing ks audio stream: {}", e.getMessage(), e);
```

Comment:
Full stack trace with a WARN level if the stream failed to close us maybe a little too much?